### ISSUE TITLE
Rework ACP module with acpmagic

### ISSUE THREAD

**Issue by Wolfsblvt** [Wolfsblvt/advancedpolls#43]
I am currently in the development of a tool wich will make creation of an ACP module much easier. I will add this to this extension as well. Just to keep that up to date. so `advancedpolls_module.php` does not need any rework at the current place.

**Comment by javiexin** [comment]
It might need just reordering and grouping of options.

**Comment by Wolfsblvt** [comment]
Yeah, that of course, but this is another point. The two methods inside that acp module are very big and long, so no good design. And I am already working on that tool for ACP, so I have created this issue as a reminder :P

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. This project is no longer maintained, so I’m closing all remaining issues as part of archiving the repository. This avoids leaving a false impression that further development or support is planned.

Anyone interested in continuing development is welcome to do so through a fork, subject to the repository’s license and any applicable third-party licenses.

**Comment by Wolfsblvt** [comment]
Thanks for reporting this and for using the project. This repository is no longer maintained and is being archived, so I’m closing all remaining issues rather than leaving them open with the implication of future development or support. See the [Disclaimer](https://github.com/Wolfsblvt/advancedpolls#disclaimer) for details.
